ce/api/multistats.py

```diff
@@ -4,7 +4,7 @@
 from ce.api.stats import stats
 from ce.api.util import search_for_unique_ids
 
-def multistats(sesh, ensemble_name='ce', model='', emission='', time=0,
+def multistats(sesh, ensemble_name='ce_files', model='', emission='', time=0,
```

Comment:
Is 0 an invalid default argument? Is this one of the queries that will return an error if it actually receives a 0 here?

Reply:
Ah right.  So that issue occurs in `stats.py`:
```
    ...
    if time: #HERE
        try:
            time = int(time)
        except ValueError:
            raise Exception('time parameter "{}" not convertable to an integer.'
                            .format(time))
    else:
        time = None
    ...
    query = sesh.query(Time.timestep).filter(Time.time_set_id == df.timeset.id)
    if time: #HERE
        query.filter(Time.time_idx == time)
    timevals = [ t for t, in query.all() ]
    timeval = mean_datetime(timevals)
    ...
```
I think changing `if time` to `if time is not None` should take care of the problem?  Since we are already converting it to an int, it shouldn't matter is we pass in `'0'` or `0`.

Reply:
Sounds good. Can you add that change to this PR?

Reply:
After further discussion, we realized that making this change would be complicated enough to warrant a separate PR.